packages/react-static/src/static/webpack/webpack.config.prod.js

```diff
@@ -3,10 +3,9 @@ import path from 'path'
 import CaseSensitivePathsPlugin from 'case-sensitive-paths-webpack-plugin'
 import { BundleAnalyzerPlugin } from 'webpack-bundle-analyzer'
 import TerserPlugin from 'terser-webpack-plugin'
-import nodeExternals from 'webpack-node-externals'
 import ExtractCssChunks from 'extract-css-chunks-webpack-plugin'
 import OptimizeCSSAssetsPlugin from 'optimize-css-assets-webpack-plugin'
-import resolveFrom from 'resolve-from'
+import { silent as resolveFrom } from 'resolve-from'
```

Comment:
I assume this change is safe because we can assume that the listed alises in `resolve.alias` that use this utility, namely `react`, `react-dom`, and `react-universal-component`, are all assumed to be installed